packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -844,7 +844,7 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       gotoValueError: '',
 
       gotoValueSelectedColumnName: model.columns[0]?.name ?? '',
-      gotoValueSelectedFilter: FilterType.eq,
+      gotoValueSelectedFilter: FilterType.eqIgnoreCase,
```

Comment:
Did @dsmmcken say we should update the default selected filter?

Reply:
Yea I might have misinterpreted what he meant but I thought whenever we search something in the UI the default should be case insensitive. If we're querying for something though using the query language that should be case sensitive.

Reply:
Yeah. Equals case insensitive as the default would be more consistent with the rest of our ui